### DISCUSSION TITLE
[@types/semver] Missing includePrerelease option in CoerceOptions

### DISCUSSION THREAD

**Discussion by christophecanel** [DefinitelyTyped/DefinitelyTyped#68673] [Issues with a @types package]
In according to the SemVer documentation, the coerce function can have a includePrerelease boolean option.
The interface ```CoerceOptions``` must therefore have a ```includePrerelease?: boolean | undefined``` property.

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in PR #68734
